public/js/templates.js

```diff
@@ -0,0 +1,61 @@
+/*
+* Copyright (c) 2021 Ecore. All rights reserved.
+*
+* University:		 Frankfurt University of Applied Sciences
+* Study program:	 Engineering Business Information Systems
+* Module:		   Advanced Programming 2021
+* Professor:		   Prof. Dr. Jung, Prof. Dr. Bremm
+* Date:			     21.04.2021
+*
+*/
+
+/**
+* A short description what this file/class is all about.
```

Comment:
Remove this sentence and put the description here. Also make sure the lines are not longer than 80 characters.